webpack.config.base.js

```diff
@@ -14,7 +14,7 @@ const webpack = require('webpack');
 const VERSION = childProcess.execSync('git rev-parse --short HEAD').toString();
 const isProduction = process.env.NODE_ENV === 'production';
 const isTest = process.env.NODE_ENV === 'test';
-const isRelease = process.env.CIRCLE_BRANCH && process.env.CIRCLE_BRANCH.startsWith('release-');
+const isRelease = process.env.GITHUB_WORKFLOW && process.env.GITHUB_WORKFLOW.startsWith('release');
```

Comment:
Just FYI this will pick up also `release-mas`

Reply:
Yep that's what we want. It should only show for nighties and dev builds.